linking.md

```diff
@@ -0,0 +1,24 @@
+# Linking
+
+## Use "evergreen" links for code
+
+When linking to code on GitHub, you can press the "y" key to obtain an "evergreen" link (ie. one that includes a SHA-1 hash instead of a symbolic name like `master`).
+
+We recommend that you favor such "evergreen" links for use in documentation and also within code comments that contain links, because "evergreen" links never break (unless history is rewritten, which we generally avoid). This is in contrast to normal links, which can totally break (ie. lead to a 404) when a file is moved, or produce confusion when lines are moved within a file.
```

Comment:
I was wondering where the term "evergreen" was coming from... I guess it makes sense from this point of view (always green, never break...)

For some reason, I was thinking about it in terms of "always up-to-the-latest" as in "evergreen browsers"... 🤔

Reply:
If it is confusing, we can change the term to "fixed". Would that be better?

(FWIW, "evergreen" actually refers to deciduous trees... the leaves never fall, even in autumn...)

Reply:
I think `fixed` is more obvious, as they're frozen in time?